crates/quickjs-wasm-rs/src/lib.rs

```diff
@@ -66,6 +66,8 @@ mod js_binding;
 mod js_value;
 mod serialize;
 
+pub use quickjs_wasm_sys;
```

Comment:
Could you add a comment regarding the safety of using this re-export?

Reply:
Is using this export inherently unsafe? All the functions by bindgen are already marked unsafe, right?

Or do you want the same comment here as above that the re-export might disappear in the future?

Reply:
Sorry yeah, not safety, but around maintenance and it might disappear in the future.

Reply:
As it turns out, doc-strings are not really a thing for re-exports (at least not when I check `cargo doc` locally). I hope the feature gate addresses your concern?

Reply:
It does, thanks!